bin/sozo/src/commands/verify.rs

```diff
@@ -0,0 +1,17 @@
+use anyhow::Result;
+use clap::Args;
+use scarb::core::Config;
+use sozo_walnut::WalnutDebugger;
+
+#[derive(Debug, Args)]
+pub struct VerifyArgs {}
+
+impl VerifyArgs {
```

Comment:
Let's also have this one walnut specific. May be into the walnut crate actually. 👍